RoomsOfRequirement/RoomsOfRequirement/__init__.py

```diff
@@ -0,0 +1,92 @@
+import sqlite3
+from sqlite3 import Error
+
+
+def create_connection(db_file):
+    """ create a database connection to the SQLite database
+        specified by the db_file
+    :param db_file: database file
+    :return: Connection object or None
+    """
+    try:
+        conn = sqlite3.connect(db_file)
+        return conn
+    except Error as e:
+        print(e)
+
+    return None
+
+def select_all(conn, table_name):
+    """
+    Query all rows in the tasks table
+    :param conn: the Connection object
+    :return:
+    """
+    cur = conn.cursor()
+    cur.execute("SELECT * FROM " + table_name)
+
+    rows = cur.fetchall()

```

Comment:
```suggestion
    rows = cur.fetchall()  # Unused variable 'rows'
```
What is this rows variable used for? Does this table return or process these rows in any way?

Reply:
Ah it's useless. Was just teaching myself how to use fetchall() and select in python